zwave_js_server/model/controller/__init__.py

```diff
@@ -691,6 +692,17 @@ async def async_get_known_lifeline_routes(
             for node_id, lifeline_routes in data["routes"].items()
         }
 
+    async def async_interview_node(self, node: Node) -> None:
```

Comment:
Z-Wave JS puts this command on the driver. Why does it do that and we put it on the controller?

Also, it says that it's not recommended to await the command. Should we use the other method to send the command, so we don't wait for a result?

https://zwave-js.github.io/node-zwave-js/#/api/driver?id=interviewnode

Reply:
whoops, that was a mistake, thanks for catching that!

Reply:
in the server, we don't await the command, we just return a response. The purpose of waiting is to make sure that we receive a response to confirm that the command was issued

Reply:
Ok.